epoch_tracker/epoch_tracker.h

```diff
@@ -305,6 +305,7 @@ class EpochTracker {
   // elements of the resid_peaks_ array of structures.
   void GetPulseCorrelations(float window_dur, float peak_thresh);
 
+ public:
```

Comment:
Do you mind moving these three methods to the public area, next to `Init` for example.

Thanks